index.js

```diff
@@ -1,18 +1,50 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
-
+//Had to change the index.js script to defer afterwards
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "blue"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+//Available because linked in index.html file with: <script src="data.js"></script>
+const travName = document.querySelector('h2')
+const travNick = document.querySelector('em')
+const travPic = document.querySelector('img')
+travName.textContent = traveler.name
+travNick.textContent = traveler.nickname
+travPic.src = traveler.photo
+
+//I'm still trying to figure out how I got here, trial and error
 
 
 /***** Deliverable 4 *****/
 
+//Iterating over animalSightings nested in traveler
+//creating a new li in the html.index
+//appended the inner HTML to update dynamically
+
+traveler.animalSightings.forEach(function(object){
+    const outerDiv = document.createElement('li')
+
+
+    outerDiv.innerHTML = `
+    <li data-id="${object.id}">
+  <p>${object.description}</p>
+  <img src="${object.photo}" alt="${object.species}"/>
+  <a href="${object.link}" target="_blank">Here's a video about the ${object.species} species!</a>
+</li>`
+
+const collection = document.querySelector('#animals')
+collection.append(outerDiv)
+
+
+})
```

Comment:
good. Be careful with variable names.

Would prefer the `outerDiv` to be renamed to `animalSightingLI` because it is an LI and not a div.

since you are using innerHTML for the LI. You are actually nesting another LI inside of it here. You can set the data-id attribute before setting the innerHTML.

```
traveler.animalSightings.forEach(obj => {
    const animalSightingsLI = document.createElement('li')
    animalSightingsLI.dataset.id = obj.id

    animalSightingsLI.innerHTML = `
        <p>${obj.description}</p>
        <img src="${obj.photo}" alt="${obj.species}"/>
        <a href="${obj.link}" target="_blank">Here's a video about the ${obj.species} species!</a>
    `
    document.querySelector('ul#animals').append(animalSightingsLI)
})
```